src/core/font_substitutions.js

```diff
@@ -0,0 +1,271 @@
+    },
+  ],
+  [
+    "Helvetica",
+    {
+      local: [
+        "Helvetica",
+        "Helvetica Neue",
+        "Arial",
+        "Arial Nova",
+        "Liberation Sans",
+        "Nimbus Sans",
+        "Nimbus Sans L",
+        "A030",
+        "TeX Gyre Heros",
+        "FreeSans",
+        "DejaVu Sans",
+        "Arimo",
+        "Albany",
+        "Bitstream Vera Sans",
+        "Arial Unicode MS",
+        "Microsoft Sans Serif",
+        "Apple Symbols",
+        "Cantarell",
+      ],
+      path: "LiberationSans-Regular.ttf",
+      style: {
+        style: "normal",
+        weight: "normal",
+      },
+      ultimate: "sans-serif",
+      baseName: "helvetica",
+    },
+  ],
+  [
+    "Helvetica-Bold",
+    {
+      local: {
+        alias: "Helvetica",
+        append: " Bold",
+      },
+      path: "LiberationSans-Bold.ttf",
+      style: {
+        style: "normal",
+        weight: "bold",
+      },
+      ultimate: "sans-serif",
+      baseName: "helvetica",
+    },
+  ],
+  [
+    "Helvetica-Oblique",
+    {
+      local: {
+        alias: "Helvetica",
+        append: " Italic",
+      },
+      path: "LiberationSans-Italic.ttf",
+      style: {
+        style: "italic",
+        weight: "normal",
+      },
+      ultimate: "sans-serif",
+      baseName: "helvetica",
+    },
+  ],
+  [
+    "Helvetica-BoldOblique",
+    {
+      local: {
+        alias: "Helvetica",
+        append: " Bold Italic",
+      },
+      path: "LiberationSans-BoldItalic.ttf",
+      style: {
+        style: "itealic",
+        weight: "bold",
+      },
+      ultimate: "sans-serif",
+      baseName: "helvetica",
+    },
+  ],
+  [
+    "Courier",
+    {
+      local: [
+        "Courier",
+        "Courier New",
+        "Liberation Mono",
+        "Nimbus Mono",
+        "Nimbus Mono L",
+        "Cousine",
+        "Cumberland",
+        "TeX Gyre Cursor",
+        "FreeMono",
+      ],
+      style: {
+        style: "normal",
+        weight: "normal",
+      },
+      ultimate: "monospace",
+      baseName: "courier",
+    },
+  ],
+  [
+    "Courier-Bold",
+    {
+      local: {
+        alias: "Courier",
+        append: " Bold",
+      },
+      style: {
+        style: "normal",
+        weight: "bold",
+      },
+      ultimate: "monospace",
+      baseName: "courier",
+    },
+  ],
+  [
+    "Courier-Oblique",
+    {
+      local: {
+        alias: "Courier",
+        append: " Italic",
+      },
+      style: {
+        style: "italic",
+        weight: "normal",
+      },
+      ultimate: "monospace",
+      baseName: "courier",
+    },
+  ],
+  [
+    "Courier-BoldOblique",
+    {
+      local: {
+        alias: "Courier",
+        append: " Bold Italic",
+      },
+      style: {
+        style: "italic",
+        weight: "bold",
+      },
+      ultimate: "monospace",
+      baseName: "courier",
+    },
+  ],
+]);
+
+function makeLocal(local) {
+  let append = "";
+  if (!Array.isArray(local)) {
+    append = local.append;
```

Comment:
Can we please add the leading space here, instead of having to remember do that manually at each entry in the list above?
```suggestion
    append = ` ${local.append};
```